src/main/java/org/opensearch/neuralsearch/processor/ScoreNormalizationTechnique.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.neuralsearch.processor;
+
+import lombok.Builder;
+import lombok.Data;
+
+import com.google.common.primitives.Floats;
+
+/**
+ * Collection of techniques for score normalization
+ */
+public enum ScoreNormalizationTechnique {
```

Comment:
Nit. Could be better to move enums to its own package?

Reply:
I would left it for later development, for now ```processor``` package looks enough